db/seeds.rb

```diff
@@ -0,0 +1,29 @@
+def generate_shoes
+  shoes = {
+    "La Sportiva" => ["Solution", "Solution Women's", 'Genius', "Testarossa"],
+    "Five Ten" => ["Team 5.10", "Blackwing", "Anasazi VCS", 'Dragon'],
+    "Evolv" => ["Shaman"],
+    "Scarpa" => ["Instinct VS"],
+    "Mad Rock" => ["Shark 2.0"],
+    "Tenaya" => ["Oasi"]
+  }
+
+  shoes.each_key do |brand|
+    shoes[brand].each do |model|
+      Shoe.create(brand: brand, model: model, size: 42)
+    end
+  end
+
```

Comment:
`Shoe.create` can fail without raising an exception so you either want to user `Shoe.create!(…)` or wrap the create in an if statement and handle the creation failure case